eev.js

```diff
@@ -59,12 +59,12 @@ var Eev = (function () {
     off: function (names, fn) {
       var me = this;
       names.split(splitter).forEach(function (name) {
-        var list = me.events[name];
-        var link = list.reg[fn._eev];
-
-        list.reg[fn._eev] = undefined;
-
-        list && link && list.remove(link);
+        if (typeof fn === 'function') {
```

Comment:
This doesn't seem to handle what I suspect would be the more common case, which is that `fn` is a function, but just not a registered event handler.

```js
fn && names.split(splitter).forEach(function (name) {
        var list = me.events[name];

        if (!list) {
          return;
        }

        var link = list.reg[fn._eev];

        list.reg[fn._eev] = undefined;

        list && link && list.remove(link);
      });
```

The above code seems to work. It ensures that there is a value `fn` and it exits if the function isn't a registered event handler.